src/libc/time.rs

```diff
@@ -268,13 +268,19 @@ struct timeval {
 }
 unsafe impl SafeRead for timeval {}
 
+#[allow(non_camel_case_types)]
+#[repr(C, packed)]
+struct timezone {
+    tz_minuteswest: i32,
+    tz_dsttime: i32,
+}
+unsafe impl SafeRead for timezone {}
+
 fn gettimeofday(
     env: &mut Environment,
     timeval_ptr: MutPtr<timeval>,
-    timezone_ptr: MutVoidPtr, // deprecated, always NULL
+    timezone_ptr: MutPtr<timezone>,
 ) -> i32 {
-    assert!(timezone_ptr.is_null());
-
     if timeval_ptr.is_null() {
         return 0; // success
```

Comment:
Oh, this needs to be changed now (don't skip setting the timezone_ptr if the timeval_ptr is null)

Reply:
Done